### PR TITLE
Added fsi/translatable integration

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open

File: composer.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
         "doctrine/persistence": "^2.3",
         "doctrine/orm": "^2.8",
         "fsi/data": "^1.0@dev",
+        "fsi/translatable": "dev-main",
```
Comment (chives) [inline composer.json:28]: we should add `branch-alias` to `fsi/translatable` to avoid such "version"

**Review by chives — APPROVED** [review]

**Comment by szymach** [comment]
@fsi-open

**Review by chives — APPROVED** [review]

**Comment by szymach** [comment]
@rn0

**Review by rn0 — APPROVED** [review]

**Pull request merged**